pkg/sanity/sanity.go

```diff
@@ -156,18 +170,35 @@ func (sc *SanityContext) teardown() {
 	// (https://github.com/kubernetes-csi/csi-test/pull/98).
 }
 
-func createMountTargetLocation(targetPath string) error {
-	fileInfo, err := os.Stat(targetPath)
-	if err != nil && os.IsNotExist(err) {
-		return os.MkdirAll(targetPath, 0755)
-	} else if err != nil {
-		return err
+func createMountTargetLocation(targetPath string, customCreateDir func(string) (string, error)) (string, error) {
+
+	// Return the target path if empty.
+	if len(targetPath) < 0 {
```

Comment:
An empty string has length 0, so this check is not getting triggered as intended.

Better compare with `if targetPath == ""`. It's more obvious what's meant (always most important) and there's no obvious performance benefit from the length comparison (performance doesn't matter here anyway).

Reply:
Sorry, that's really a mistake :sweat_smile: . Changed it.